integration-libs/s4om/src/s4om.module.ts

```diff
@@ -0,0 +1,8 @@
+import { NgModule } from '@angular/core';
+import { ScheduleLinesModule } from './schedule-lines';
```

Comment:
Also, looks like src dir is missing index and public api files

Reply:
agreed, will be adding the same.